vmaas/reposcan/database/csaf_store.py

```diff
@@ -46,7 +48,129 @@ def _save_csaf_files(self, csaf_files: CsafFiles) -> list[int]:
         self.conn.commit()
         return db_ids
 
+    def _get_product_rows(self, csaf_cves):
+        to_insert = []
+        to_update = None
```

Comment:
remove it from here since it is never used and just explicitly return `None` in return statement as `return to_insert, None`

Reply:
I did it on purpose. I like the information that this function returns no rows to update.

Reply:
yes so return `return to_insert, None` 😄 so you don't even need to read the function and you know it returns None, but if you have strong preference then it's fine as it is, functionality is the same